src/compiler/parser.ts

```diff
@@ -6996,7 +6996,7 @@ namespace ts {
         }
 
         function parseNamespaceExport(pos: number): NamespaceExport {
-            return finishNode(factory.createNamespaceExport(parseIdentifier()), pos);
+            return finishNode(factory.createNamespaceExport(parseIdentifierName()), pos);

```

Comment:
what does this change do?

Reply:
An *IdentifierName* is any valid user-declarable *Identifier* along with any reserved keywords.

Reply:
That's actually backwards: An _Identifier_ is any valid _IdentifierName_ **excluding** reserved keywords:

[![Identifier Production](https://user-images.githubusercontent.com/3902892/88864977-6ca40700-d1bb-11ea-8158-ca5f5d7e1302.png "Identifier: IdentifierName but not ReservedWord")](https://tc39.es/ecma262/#prod-Identifier)

Reply:
tomato / tomat + 0

Reply:
or maybe

tomato / tomatoo - o